server/errors.json

```diff
@@ -1298,5 +1298,15 @@
     "help": "",
     "url": "",
     "deprecates": ""
+  },
+  {
+    "constant": "JSConsumerCreateDurableAndNameMismatch",
+    "code": 400,
+    "error_code": 10132,
+    "description": "Consumer Durable and Name has to be equal if both provided",
```

Comment:
nit: should it be "have to be" instead of "has to be" since we are talking about Durable and Name? (take it from a non native so really not sure :-) )

Reply:
`should be`